tests/nn/data_parallel/test_fsdp_regnet.py

```diff
@@ -215,7 +334,10 @@ def test1(temp_files, ddp_ref, precision, flatten):
     fsdp_config["mixed_precision"] = precision == "mixed"
     fsdp_config["flatten_parameters"] = flatten == "flatten"
 
-    world_size = 2
+    if fsdp_config["mixed_precision"] and torch_cuda_version() < (11, 0):
```

Comment:
@msbaines It is great that CI has a CUDA 11 VM. Otherwise, this test would have no place to run. It seems that we should gradually move everything to CUDA 11.